scenarios/test_codegen/schema.graphql

```diff
@@ -120,6 +120,7 @@ type EntityWithAllTypes {
   # optTimestamp: Timestamp
   # NOTE: array of timestamps has the same problem as array of booleans described above
   # arrayOfTimestamps: [Timestamp!]!
+  json: Json!
```

Comment:
This doesn't break this test? https://github.com/enviodev/hyperindex/pull/525

Reply:
I didn't merge the change yet, but for the JSON type it shouldn't use unnest.